ts/Stock/Navigator/Navigator.ts

```diff
@@ -1558,6 +1558,7 @@ class Navigator {
                 showInLegend: false,
                 stacking: void 0, // #4823
                 isInternal: true,
+                isNavigator: true, // #17212
```

Comment:
I think you can just use the `isInternal` property, instead of adding a new property. It is mostly used by the navigator.

Reply:
Thank you, I had no idea about it :)